src/conv-core/cputopology.C

```diff
@@ -198,6 +196,32 @@ static void printTopology(int numNodes)
     CmiPrintf("Charm++> Running on %d hosts\n", numNodes);
 }
 
+static std::atomic<char> cpuTopoSyncHandlerDone;
+#if CMK_SMP && !CMK_SMP_NO_COMMTHD
+extern void CommunicationServerThread(int sleepTime);
+static std::atomic<char> cpuTopoSyncCommThreadDone;
```

Comment:
Is there a reason this is not a `std::atomic<bool>`? Should this be explicitly initialized? (same  question for cpuTopoSyncHandlerDone)

Reply:
Done